playground/backend/internal/fs_tool/java_fs.go

```diff
@@ -0,0 +1,140 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fs_tool
+
+import (
+	"fmt"
+	"github.com/google/uuid"
+	"io/fs"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+const (
+	parentBaseFileFolder   = "internal"
+	javaBaseFileFolder     = parentBaseFileFolder + "/executable_files"
+	javaSrcFileFolder      = javaBaseFileFolder + "/src"
+	javaBinFileFolder      = javaBaseFileFolder + "/bin"
+	javaExecutableFileType = "java"
+	javaCompiledFileType   = "class"
+)
+
```

Comment:
In addition to comments on the interface in playground/backend/internal/fs_tool/fs.go:

```
var (

  javaGlobs = []string{
       parentBaseFileFolder,
       javaBaseFileFolder,
       javaSrcFileFolder,
       javaBinFileFolder,
  }

   javaFs = os.DirFS(parentBaseFileFolder)
)

```